cmd/warcserver/cmd/index/io.go

```diff
@@ -51,7 +51,10 @@ func ReadFile(c *conf, writer index.CdxWriter) error {
 		}
 		count++
 
-		writer.Write(wr, c.fileName, currentOffset)
+		err = writer.Write(wr, c.fileName, currentOffset)
+		if err != nil {
+			log.Warnf("Failed to writer to %s: %v", c.fileName, err)
```

Comment:
Why not return error?

Reply:
If you would like to continue writing i think warning is the correct log level, but the log message must provide `currentOffset` and `count` to be useful.

Reply:
Done